### ISSUE TITLE
MLML problems

### ISSUE THREAD

**Issue by soleilll** [smithlabcode/methpipe#144]
hello！
I have only two types of input are available,so i  use them  with the following command:
mlml -u M7_RRBS.meth -m M7_oxRRBS.sort.meth -o test

but it said “mlml: mlml.cpp:509: int main(int, const char**): Assertion `f_chr == s_chr && f_pos == s_pos' failed.Aborted (core dumped)”

So would you please help me with these error？thank you very much！！！

**Comment by terencewtli** [comment]
Hi! Could you give us some guidance so we can reproduce the error? If you can find the line in the input files that give this error, that would be great. Running ```head -n 100 your_file``` on both input files might help too.

**Comment by terencewtli** [comment]
I've updated the code to throw a runtime error. Could you run it and show us the output?

**Comment by terencewtli** [comment]
Right now, we suspect that the files are not sorted the same way. Could you confirm that both files are sorted using the same locale?

**Comment by Soleilcode** [comment]
@terencewtli 
Hi！ I'm sorry for the late reply. I have been delayed in some matters these days.
I checked my files.I downloaded the test file and I generated a new file in the format of this file. Then sorted them with the code `sort -k1,1 -k2,2n my_file`. Finally,it works! But i forgot to filter my data with the coverage, so i got many 100% hmc. Anyway, it is another question. I have solved this mlml problem already.
Thank you for your advise!!!
